fairlearn/metrics/demographic_parity.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+class DemographicParity:
```

Comment:
Do we want to have a base class FairnessMetric that all of these inherit from?

Reply:
I have been thinking about that - particularly given the composability we've discussed as an idea. Having said that....  to what extent do base classes have meaning in Python? Unless there is common functionality, I'm not sure there's a huge amount of value.